tests/integration/targets/lookup_dig/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Install dnspython library
+  pip:
+    name: dnspython
+    state: present
```

Comment:
Please add a `meta/main.yml` with
```
dependencies:
  - setup_remote_constraints
```
and then add `extra_args: "-c {{ remote_constraints }}"` to this task. Sooner or later we might have to add constraints, and that is a lot easier if this task also uses the one from `tests/utils/constraints.txt`.